tests/framework/microvm.py

```diff
@@ -248,7 +249,7 @@ def __init__(
         self.disks_vhost_user = {}
         self.vcpus_count = None
         self.mem_size_bytes = None
-        self.cpu_template_name = None
+        self.cpu_template_name = "None"
```

Comment:
nit: why cant we just store the CPU template, and call `cpu_template_name(self.cpu_template)` whenever we need it?

Reply:
yeah, I also don't like the current implementation, but I'd like to keep the change minimal in this PR. In any case, I'll do much refactoring (including simplifying code around this) later since I found some bugs.